packages/kbn-securitysolution-exceptions-common/api/find_exception_list/find_exception_list.gen.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Find exception lists API endpoint
+ *   version: 2023-10-31
+ */
+
+import { z } from 'zod';
+import { ArrayFromString } from '@kbn/zod-helpers';
+
+import { NonEmptyString } from '@kbn/openapi-common/schemas/primitives.gen';
+import { ExceptionNamespaceType, ExceptionList } from '../model/exception_list_common.gen';
+
+export type FindExceptionListsFilter = z.infer<typeof FindExceptionListsFilter>;
+export const FindExceptionListsFilter = NonEmptyString;
+
+export type FindExceptionListsRequestQuery = z.infer<typeof FindExceptionListsRequestQuery>;
+export const FindExceptionListsRequestQuery = z.object({
+  /** 
+      * Filters the returned results according to the value of the specified field.
+
+Uses the `so type.field name:field` value syntax, where `so type` can be:
+
+- `exception-list`: Specify a space-aware exception list.
+- `exception-list-agnostic`: Specify an exception list that is shared across spaces.
+ 
+      */
+  filter: FindExceptionListsFilter.optional(),
+  /** 
+      * Determines whether the returned containers are Kibana associated with a Kibana space
+or available in all spaces (`agnostic` or `single`)
+ 
+      */
+  namespace_type: ArrayFromString(ExceptionNamespaceType).optional().default(['single']),
+  /**
+   * The page number to return
+   */
+  page: z.coerce.number().int().min(0).optional(),
```

Comment:
According to `packages/kbn-securitysolution-io-ts-types/src/string_to_positive_number/index.ts` both `page` and `per_page` should be positive numbers. We use `StringToPositiveNumber` type which does failure check `stringAsNumber <= 0`. Can we use `z.number().positive()` here?

Reply:
Same in other files where we use these types.

Reply:
I fixed it in two files

- `packages/kbn-securitysolution-exceptions-common/api/find_exception_list/find_exception_list.schema.yaml`
- `packages/kbn-securitysolution-exceptions-common/api/find_exception_list_item/find_exception_list_item.schema.yaml`

I don't see any other `page` and `per_page` fields definitions.